blackbox/docs/sql/statements/copy-from.rst

```diff
@@ -35,15 +35,33 @@ Description
 import.
 
 The nodes in the cluster will attempt to read the files available at the URI
-and import the data. These files have to be UTF-8 encoded and contain a single
-JSON object per line. Any keys in the object will be added as columns,
-regardless of the previously defined table. Empty lines are simply sikpped.
+and import the data.
 
+Supported Formats
+-----------------
+CrateDB accepts both JSON and CSV inputs. This can be specified using either the
+relevant file extension, or by providing the file type as an option (see ``WITH``
+below), which may be useful if the files do not have a `.json` or `.csv` file extension.
+In the case that a file does not have a relevant file extension and format option has not
+been provided, then by default the file with be processed as JSON.
```

Comment:
how about this:

> CrateDB accepts both JSON and CSV inputs. The format is guessed from the file extension (``.json`` or ``.csv`` respectively) if possible. The format can also be provided as an option (see \`WITH\`_). If a format is not specified and the format cannot be guessed, the file will be processed as JSON.

note: the \`WITH\`_ is an [implicit link](https://stackoverflow.com/questions/18632781/how-to-make-an-internal-link-to-a-heading-in-sphinx-restructuredtext-without-cre/18704682#18704682). but there's a chance it might not work because the section heading uses literal syntax. if that's the case, I would suggest adding [explicit section labelling](http://www.sphinx-doc.org/en/stable/markup/inline.html#role-ref) throughout the doc